codebuild/bin/install_default_dependencies.sh

```diff
@@ -58,6 +58,11 @@ if [[ "$S2N_LIBCRYPTO" == "boringssl" && ! -d "$BORINGSSL_INSTALL_DIR" ]]; then
     codebuild/bin/install_boringssl.sh "$(mktemp -d)" "$BORINGSSL_INSTALL_DIR" > /dev/null ;
 fi
 
+# Download and Install AWS-LC
+if [[ "$S2N_LIBCRYPTO" == "awslc" && ! -d "$AWSLC_INSTALL_DIR" ]]; then
```

Comment:
non-blocker: I am a fan of guarding variable expansions.

Reply:
s2n does not use guarding variable expansion. To make code syntax consistent, this change may need to be in another PR.